pkgs/development/python-modules/scrapy/default.nix

```diff
@@ -0,0 +1,36 @@
+{ self, buildPythonPackage, pkgs, lib }:
```

Comment:
- never pass in `pkgs`
- python libraries are passed the individual python packages, not pythonPackages or self